kratos/sources/model_part.cpp

```diff
@@ -1786,6 +1786,9 @@ void ModelPart::PrintData(std::ostream& rOStream) const
     rOStream << "    Number of tables : " << NumberOfTables() << std::endl;
     rOStream << "    Number of sub model parts : " << NumberOfSubModelParts() << std::endl;
     if (!IsSubModelPart()) {
+        if (IsDistributed()) {
+        rOStream << "    Distributed over " << mpCommunicator->TotalProcesses() << " processes" << std::endl;
```

Comment:
@adityaghantasala this is the size of the `MPI_Comm` with which the data-communicator was created
By default this is the size of `MPI_COMM_WORLD`, but with a sub-communicator it would be the size of this sub-communicator

Reply:
Ya exactly.